### ISSUE TITLE
new functionnality

### ISSUE THREAD

**Issue by yohji79** [pfitzer/kunena-discord#9]
is it possible to add a fonctionnality, to have the possibility to have a webhook for a kunena categorie, in order to connect more channel with precise category.

do you think it's possible ?

**Comment by pfitzer** [comment]
This is a thing i allready thought about and yes it is possible. My idea is to give the ability to choose _n_ categories for a webhook. Only problem is the fixed amount of webhooks, cause it is not possible to make the count of webhooks in the settings dynamic. So the question is, how many possible webhooks to we need? 5, 10 or even more?

**Comment by yohji79** [comment]
They didn't talk about limitation here in the documentation 
https://discordapp.com/developers/docs/resources/webhook
Perhaps there isn't limitation, after all even if It's 10 to start it's a good branche to test

**Comment by pfitzer** [comment]
The limitation is not in discord, the problem is how joomla handles plugin settings. We are forced to use a fixed amount of possible webhooks. I think 10 is a good starting point, too,

**Comment by yohji79** [comment]
Yes, try it, we will see
The same plug-ins for iCalendar component could be also really cool

**Comment by pfitzer** [comment]
Do you have a link to the extension homepage?

**Comment by yohji79** [comment]
Here https://icagenda.joomlic.com/

**Comment by pfitzer** [comment]
Can you please check the french translations in this branch https://github.com/pfitzer/kunena-discord/tree/issue%239

And if you wanna make some tests with the new version
[kunena-discord_2.0.zip](https://github.com/pfitzer/kunena-discord/files/4564316/kunena-discord_2.0.zip)

You can set 10 different Webhooks and choose from the desired categories.

**Comment by yohji79** [comment]
French translation is Good, 
I'll make a try with the new version :) 
I would like to program as good as you, I have a project but my level is too bad :)

**Comment by yohji79** [comment]
got an 404 error when I make test 
original link
http://www.blacksheep-squadron.fr/index.php/kunena-2018-12-17/briefing-de-mission/430-test-briefing
back link
https://www.blacksheep-squadron.fr//index.php/kunena-2018-12-17/briefing-de-mission/430-test-briefing#967
there is a double // after the domain name

**Comment by pfitzer** [comment]
I have nothing changed on link generation. If you set the url in plugin settings, use it without / at the end. Maybe this is the problem.

Starting programming is not always easy. And if you start programming PHP especially with Joomla it could be very frustrating. Joomla is not the best framework or CMS to start with learning PHP.  Joomla follows the rule: why simple if you can make it complicated ;) And its design is outdated.

But, keep on practicing and keep on turning coffee into code and you get better.

**Comment by yohji79** [comment]
Problem Solved :) thank you

new functinnality works well and it's a good way to mix forum and discord interactions

**Comment by pfitzer** [comment]
I took a look at the iCagenda component and unfortunately, there is no integrated system to dispatch events when a new entry is added/updated like in Kunena. I found a "iCagendaOnNewEvent" event, but this is fired in strange situations. At the moment i don´t know if this will work to push do Discord.
I´m very busy at the moment, but if i find the time and be in the mood for this, i will take a deeper look into.
